.velocitas.json

```diff
@@ -2,7 +2,7 @@
     "packages": [
         {
             "name": "devenv-runtimes",
-            "version": "v2.2.5"
```

Comment:
fix tag

Reply:
Please fix before integration :)

Reply:
that's why I added my comment :) now the correct tag is used